specs/capella/beacon-chain.md

```diff
@@ -245,7 +251,8 @@ class BeaconState(Container):
     # Execution
     latest_execution_payload_header: ExecutionPayloadHeader
     # Withdrawals
-    withdrawal_index: WithdrawalIndex
+    withdrawal_index: WithdrawalIndex  # [New in Capella]
+    next_partial_withdrawal_index: ValidatorIndex  # [New in Capella]
```

Comment:
the variable names & types look confusing here 😅 but `next_partial_withdrawal_validator_index` may be too long...

What do you think about:
1. rename `withdrawal_index` to `withdrawals_queue_index`; `WithdrawalIndex` to `WithdrawalsQueueIndex`
2. move it next to `withdrawals_queue`

Reply:
yeah, I wasn't totally happy with the name too.

suggestion looks reasonable. I'll think about that and a few other naming issues

Reply:
i think `WithdrawalQueueIndex` is actually a bit misnamed because the queue length dynamically changes.

I'm going with `next_partial_withdrawal_validator_index` for now